src/main/java/org/apache/maven/plugins/enforcer/RequireLicense.java

```diff
@@ -0,0 +1,59 @@
+package org.apache.maven.plugins.enforcer;
+
+import org.apache.maven.enforcer.rule.api.EnforcerRule;
+import org.apache.maven.enforcer.rule.api.EnforcerRuleException;
+import org.apache.maven.enforcer.rule.api.EnforcerRuleHelper;
+import org.apache.maven.model.License;
+import org.apache.maven.plugin.logging.Log;
+import org.apache.maven.project.MavenProject;
+import org.codehaus.plexus.component.configurator.expression.ExpressionEvaluationException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Manuel Recena
```

Comment:
I would recommend adding some description here

Reply:
Sure. I will add when the approach has consensus 😄